src/createServer.js

```diff
@@ -1,8 +1,44 @@
 'use strict';
 
+const http = require('http');
+const fs = require('fs');
+const url = require('url');
+
 function createServer() {
-  /* Write your code here */
-  // Return instance of http.Server class
+  return http.createServer((req, res) => {
+    res.setHeader('Content-Type', 'text/plain');
+
+    const normalizedUrl = new url.URL(req.url, `http://${req.headers.host}`);
+
+    if (!normalizedUrl.pathname.startsWith('/file')) {
+      res.statusCode = 400;
+      res.statusMessage = 'Path should start with "/file/"';
+      res.end();
+    } else {
+      const requestedPath =
+        normalizedUrl.pathname === '/file' ||
+        normalizedUrl.pathname === '/file/'
+          ? 'index.html'
+          : normalizedUrl.pathname.slice(6);
+
+      if (requestedPath.includes('//')) {
```

Comment:
The check for double slashes in the path is correct, but it might be more robust to use a regular expression to catch other invalid path sequences as well.